resteasy-core/src/main/java/org/jboss/resteasy/core/ContextParameterInjector.java

```diff
@@ -221,4 +240,150 @@ public ClassLoader run() {
       }
       return set.toArray(new Class<?>[]{});
    }
+
+   public class ContextOutputStream extends ServletOutputStream
+   {
+      private ServletOutputStream delegate;
```

Comment:
This should be final.

Reply:
Done.

Reply:
Sorry, I meant the field not that class :) It shouldn't be mutable for the sake of "self documenting code" IMO.